com.unity.render-pipelines.high-definition/Documentation~/Override-Screen-Space-GI.md

```diff
@@ -46,13 +46,16 @@ HDRP uses the [Volume](Volumes.md) framework to calculate SSGI, so to enable and
 | - **Denoiser Radius**          | Set the radius of the spatio-temporal filter.                |
 | - **Second Denoiser Pass**     | Enable this feature to process a second denoiser pass. This helps to remove noise from the effect. |
 | **Depth Tolerance** | Use the slider to control the tolerance when comparing the depth of the GameObjects on screen and the depth buffer. Because the SSR algorithm can not distinguish thin GameObjects from thick ones, this property helps trace rays behind GameObjects. The algorithm applies this property to every GameObject uniformly. |
+| **Ray Miss**         | Defines if HDRP should use the reflection probes, the sky, both or nothing as a fall-back for screen space global illumination when a ray doesn't find an intersection. |
 
 ### Ray-traced
 
 ![](Images/Override-ScreenSpaceGlobalIllumination2.png)
 
 | Property                       | Description                                                  |
 | ------------------------------ | ------------------------------------------------------------ |
+| **Ray Miss**                   | Defines if HDRP should use the reflection probes, the sky, both or nothing as a fall-back for ray-traced global illumination when a ray doesn't find an intersection. |
```

Comment:
Determines what HDRP does when ray-traced global illumination (RTGI) doesn't find an intersection. . Choose from one of the following options: <br/>
• **Reflection probes**: HDRP uses reflection probes in your scene to calculate the last RTGI bounce.<br/>
• **Sky**: HDRP uses the sky defined by the current [Volume](Volumes.md) settings to calculate the last SSGI bounce.<br/>
• **Both** : HDRP uses both reflection probes and the  the sky defined by the current [Volume](Volumes.md) settings to calculate the last SSGI bounce.<br/>
• **Nothing**: HDRP does not calculate indirect lighting when RTGI doesn't find an intersection.<br/><br/>

This property is set to **Both** by default